src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -467,6 +576,9 @@ def postprocess(self, model_outputs, decoder_kwargs: Optional[Dict] = None, retu
                 chunks.append({"text": item[return_timestamps], "timestamp": (start, stop)})
             optional["chunks"] = chunks
 
+        elif return_timestamps and self.type == "seq2seq":
+            optional["chunks"] = char_offsets
```

Comment:
Where is `char_offsets` defined ?

(+ it's not a good name).

I don't think it's a good idea to check on `type` here.
This code is meant to be orthogonal, if it cannot be then we already have an if `type` earlier which can be reused